src/make-model.js

```diff
@@ -3,7 +3,7 @@ const makeModel = (e) => { // eslint-disable-line arrow-body-style
     x: e.pageX,
     y: e.pageY,
     value: 1,
-    radius: 20
+    radius: 10
```

Comment:
I'd be cool to have different radius per different actions tracked (I will explain what I mean)